#!/usr/bin/env python
PACKAGE = "second_project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Add the parameter for remapped frame with default value "base_link"
gen.add("remapped_frame", str_t, 0, "The frame to which the PointCloud should be remapped", "base_link")

exit(gen.generate(PACKAGE, "lidar_remap", "lidar"))
